tests/tests.js

```diff
@@ -536,3 +532,48 @@ QUnit.test('escapedCharacterString', function(assert) {
     'hello &amp;amp;&amp;gt;&amp;#39;&amp;lt;&amp;amp; &amp;quot;'
   );
 });
+
+QUnit.test('unescapeHTML', function(assert) {
+  var html = '&amp;lt;div&amp;gt;&amp;lt;/div&amp;gt;';
+  var unescapedHTML = unescapeHTML(html, 2);
+  assert.equal(unescapedHTML, '<div></div>');
+});
+
+QUnit.test('minimizeStyles', function(assert) {
+  var message = {
+    'html': [
+        '<div id="myId"',
+        'style="animation-delay: 0s; width: 5px;" ',
+        '></div>'
+    ],
+    'frameHoles': null,
+    'idToStyleIndex': {"myId": 1},
+    'windowHeight': 5,
+    'windowWidth': 5,
+    'frameIndex': '0'
+  };
+  minimizeStyles(message);
+  assert.equal(message.html[1], 'style="width: 5px;" ');
+});
+
+QUnit.test('serialize tree: end-to-end', function(assert) {
```

Comment:
Test looks great!

I was surprised to see transform-origin change when the border changes, but I think it's just how the system works... Possibly transform-origin is spec'd to be the center of an object by default?